ts/components/ContextualHelpComponent.tsx

```diff
@@ -33,50 +33,85 @@ const ContextualHelpComponent: React.FunctionComponent<Props> = ({
   isContentLoaded,
   onLinkClicked,
   onRequestAssistance
-}) => (
-  <>
-    <BaseHeader
-      accessibilityEvents={{
-        avoidNavigationEventsUsage: true
-      }}
-      headerTitle={I18n.t("contextualHelp.title")}
-      customRightIcon={{
-        iconName: "io-close",
-        onPress: onClose,
-        accessibilityLabel: I18n.t("global.accessibility.contextualHelp.close")
-      }}
-    />
+}) => {
+  // Being content a ReactNode we can rely on `null` to represent no-content
+  const showLoader = contextualHelpData.content === undefined;
```

Comment:
```suggestion
  const isContentReady = contextualHelpData.content === undefined;
```
Could it be more descriptive?